crates/ra_cli/Cargo.toml

```diff
@@ -6,8 +6,10 @@ authors = ["rust-analyzer developers"]
 publish = false
 
 [dependencies]
+itertools = "0.8.0"
 pico-args = "0.3.0"
 env_logger = { version = "0.7.1", default-features = false }
+rand = { version = "0.7.0", features = ["small_rng"] }
```

Comment:
I think this is fine, but I'd love to remove deps on rand altogether one day. I don't like that we have two of them.

For our purposes, [this should be enough](https://github.com/rust-lang/rust/blob/61d9231ff2604a0467987042d9ebf9ff9ea739b5/src/libcore/slice/sort.rs#L491-L498).

(immediate action here is blocked on removing proptest in favor of quickcheck. and cc https://github.com/BurntSushi/quickcheck/issues/241)

Reply:
Using a PRNG would make it deterministic, which is exactly the opposite of what the `--randomize` option is supposed to do.

Reply:
For our purposes, seeding with current time (good old `srand(time(NULL))`) should be enough, we don't need OS-level crypto random numbers.

Reply:
I did check whether we still depended on rand, and only added it since we already depend on it :smile: I didn't really want to implement `shuffle` myself for this.